spec/Candid.md

```diff
@@ -1064,7 +1076,7 @@ Most Candid values are self-explanatory, except for references. There are two fo
 Likewise, there are two forms of Candid values for function references:
 
 * `ref(r)` indicates an opaque reference, understood only by the underlying system.
-* `pub(s,n)`, indicates the public method name `n` of the service referenced by `s`.
+* `pub(s,n,v*:t*)`, indicates the public method name `n` of the service referenced by `s`, possibly followed by a list of type-annotated bound argument values.
```

Comment:
You don’t want to support binding arguments to references?

Why not make it a recursive definition that allows you to bind arguments to any function reference (whether it’s opaque, public, or itself a closure – after all, these are all values of the same type, so binding to them should be allowed).
```suggestion
* `closure(f,v*:t*)`, indicates the function reference `f`, followed by a list of type-annotated bound argument values.
```
```